Properties/Languages2020.md

```diff
@@ -1,6 +1,6 @@
 <!-- Heading -->
 # Bootcamp 2 
+Add Coby and Cory
```

Comment:
Who is Coby?

Reply:
This is a file for takeaways, so you have to submit your takeways from Languages section.